### DISCUSSION TITLE
Renewed github student does not give github copilot access

### DISCUSSION THREAD

**Discussion by drakeerv** [community/community#137880] [GitHub Education]
### Hi everyone,

Hello! I recently renewed my github student status and I got accepted, however, when I checked vscode my copilot says that I am not authorized. Any help would be appreciated!

**Comment by pg56714** [comment]
@drakeerv 
Mark me as answer thanks

[copilot](https://github.com/features/copilot)

Click **Get started with Copilot**.

Click **Start a free trial**.

That's awesome! It really is free to use, with the big text saying, **Congratulations! You are eligible to use GitHub Copilot for free**.

![image](https://github.com/user-attachments/assets/3db0309c-4888-435a-9378-e57d850093c2)
![image](https://github.com/user-attachments/assets/bb49cfe9-4cde-460e-9503-a38b28fb3648)
![image](https://github.com/user-attachments/assets/d3fa07f3-d37f-4561-9d40-39c0f879fdf7)
